src/plots/plots.js

```diff
@@ -745,7 +745,7 @@ plots.supplyLayoutModuleDefaults = function(layoutIn, layoutOut, fullData) {
 
     // TODO register these
     // Legend must come after traces (e.g. it depends on 'barmode')
-    var moduleLayoutDefaults = ['Fx', 'Annotations', 'Shapes', 'Legend'];
+    var moduleLayoutDefaults = ['Fx', 'Annotations', 'Shapes', 'Legend', 'RangeSlider'];
```

Comment:
let's remove `RangeSlider` from that list.

The range slider defaults step should be called in the axis default step, below this [line](https://github.com/plotly/plotly.js/blob/c29e81ef1f5d34d7d1e3d4a1a9ba4b2cced68304/src/plots/cartesian/layout_defaults.js#L132).